security/oidc-client-mutual-tls/src/test/java/io/quarkus/ts/security/oidcclient/mtls/MutualTlsKeycloakService.java

```diff
@@ -9,39 +8,28 @@
 
 public class MutualTlsKeycloakService extends KeycloakService {
 
-    private static final String REALM_DEFAULT = "test-mutual-tls-realm";
-    private static final String CERT_PATH = "/etc/x509/https/";
-    private static final String MUTUAL_TLS_BUNDLE_PATH = CERT_PATH + "ca-client.bundle";
-    private static final String TLS_CRT_PATH = CERT_PATH + "tls.crt";
-    private static final String TLS_KEY_PATH = CERT_PATH + "tls.key";
-    private static final String CREATE_REALM_FILE_PATH = "/keycloak-realm.json";
     private static final String X509_CA_BUNDLE = "X509_CA_BUNDLE";
-    private final boolean openshiftScenario;
-    private final String realm;
 
-    private MutualTlsKeycloakService(String createKeycloakRealmPath, String realm) {
-        super(createKeycloakRealmPath, realm);
-        openshiftScenario = false;
-        this.realm = realm;
-    }
+    private final String realm;
+    private final String realmBasePath;
+    private final boolean openshiftScenario;
 
-    private MutualTlsKeycloakService(String realm) {
-        super(realm);
-        openshiftScenario = true;
-        this.realm = realm;
-    }
+    public static final String KC_DEFAULT_JKS_CMD_VALUE = "start-dev " +
```

Comment:
It would be nice to have some kind of explanation in the name. Since this variable is used only in context, such as this one: `@KeycloakContainer(command = KC_DEFAULT_JKS_CMD_VALUE,` I suggest to change the name to something akin to RUN_IN_DEVELOPMENT_MODE_JKS(I am not sure, that this command does, so the name is up to you)